mycroft/client/speech/main.py

```diff
@@ -41,6 +41,12 @@
 
 def handle_record_begin():
     logger.info("Begin Recording...")
+
+    # If enabled, play a wave file with a short sound to indicate recording has
+    # begun.
+    if config.get('confirm_listening'):
+        play_wav(expanduser(config.get('confirm_ding')))
```

Comment:
`expanduser` has not been imported into the file. Locally, I just removed the function call. But probably, should consider adding the import if this is a best practice or style guideline.